README.md

```diff
@@ -728,6 +729,18 @@ provides basic RuboCop integration for RubyMine/IntelliJ IDEA.
 Here's one great opportunity to contribute to RuboCop - implement
 RuboCop integration for your favorite editor.
 
+## Git pre-commit hook integration
+
+[overcommit](https://github.com/brigade/overcommit) is a fully configurable and
+extendable Git commit hook manager. To use RuboCop with overcommit, add the
+following to your `.overcommit.yml` file:
+
+```yaml
+PreCommit:
+  Rubocop:
```

Comment:
Good call out, but it isn't entirely clear to me what naming scheme overcommit uses. I've opened an issue for this at https://github.com/brigade/overcommit/issues/199.